src/jdk.jlink/share/classes/jdk/tools/jmod/resources/jmod_de.properties

```diff
@@ -57,7 +57,7 @@ main.opt.man-pages=Speicherort der Manpages
 main.opt.target-platform=Zielplattform
 main.opt.target-platform.arg=target-platform
 main.opt.module-path=Modulpfad
-main.opt.hash-modules=Berechnet und erfasst Hashes zur Bindung eines in ein Package integrierten Moduls an Module, die dem angegebenen <Regex-Muster> entsprechen und direkt oder indirekt davon abhängen. Die Hashes werden in der erstellten JMOD-Datei oder in einer JMOD- oder modularen JAR-Datei in dem Modulpfad erfasst, der im jmod-Hashbefehl angegeben ist.
+main.opt.hash-modules=Berechnet und erfasst Hashes zur Bindung eines verpackten Moduls an Module, die dem angegebenen <Regex-Muster> entsprechen und direkt oder indirekt davon abhängen. Die Hashes werden in der erstellten JMOD-Datei oder in einer JMOD- oder modularen JAR-Datei in dem Modulpfad erfasst, der im jmod-Hashbefehl angegeben ist.
```

Comment:
I assume this is more evidence that the German differences are just updated translations for German?

Reply:
Yes, there were standalone German translation updates. Some were also reverts to fixes we made last time for German translations specifically, as they were rejected by the translation team. In any case, I think it is probably best not to deviate from their translations unless they are incorrect.

Reply:
Makes sense. I agree then.